fe1-web/src/features/rollCall/components/RollCallOpen.tsx

```diff
@@ -191,6 +192,9 @@ const RollCallOpen = ({
               ]}>
               {popToken}
             </Text>
+            <PoPTextButton onPress={() => Clipboard.setStringAsync(popToken)}>
+              {STRINGS.general_copy}
+            </PoPTextButton>
```

Comment:
That works, if you want to make Text selectable you could also set that attribute https://reactnative.dev/docs/text#selectable :)